### PR TITLE
Validate that response body is not empty

### PR DESCRIPTION
**Pull request by claudiahdz**
Added a tiny verification for the data variable since some customers where seeing errors in which this variable was undefined.

ex.
The following line causes around 100 errors within 24 hours:
`for (a = 0; a < data.body.keys.length && matchingKey === null; a++) {`

The error message is:
`Cannot read property 'body' of undefined``

ZENDESK Ticket: https://auth0.zendesk.com/agent/tickets/43507

### CODE REVIEW

**Comment by luisrudge** [comment]
Can you please write a test for this case? Thanks!

**Comment by claudiahdz** [comment]
@luisrudge Corrected it and tests are passing. Is that okay?

**Comment by luisrudge** [comment]
tests are passing, but there's no new test for the behavior you just added.

**Comment by claudiahdz** [comment]
Test added.

File: src/helpers/jwks.js [hunk 1]
```diff
@@ -26,6 +26,10 @@ function getJWKS(options, cb) {
         return cb(err);
       }
 
+      if (!data.body) {
+        return cb({error: true}, data);
```
Comment (luisrudge) [inline src/helpers/jwks.js]: Considering this error will be returned to the public API method (and then returned to auth0.js), we should have a more descriptive error message.

https://github.com/auth0/idtoken-verifier/blob/d30aa9310ac2f6a5ac1412a587ce062cc844013d/src/index.js#L201

**Comment by cocojoe** [comment]
Approved the closed?

**Pull request closed without merging**